src/main/scala/io/iohk/ethereum/consensus/pow/difficulty/EthashDifficultyCalculator.scala

```diff
@@ -9,6 +9,7 @@ object EthashDifficultyCalculator extends DifficultyCalculator {
   private val ExpDifficultyPeriod: Int = 100000
   private val ByzantiumRelaxDifficulty: BigInt = 3000000
   private val ConstantinopleRelaxDifficulty: BigInt = 5000000
+  private val MuirGlacierRelaxDifficulty: BigInt = 9000000
```

Comment:
Just for the sake of lisibility (and because I have bad eyes), wouldn't be nice to use the 9_000_000 syntax for all those multiple-zeros numbers ?